ssg/constants.py

```diff
@@ -75,6 +75,33 @@
     "Verify Ansible meets SCAP-Security-Guide version requirements."
 standard_profiles = ['standard', 'pci-dss', 'desktop', 'server']
 
+
+OVAL_SUB_NS = dict(
+    ind="independent",
+    unix="unix",
+    linux="linux",
+)
+
+
+ALL_NS = {
```

Comment:
How about following pattern in the file and name it something like `PREFIX_to_NS`?